public/app/plugins/datasource/grafana-azure-monitor-datasource/types/query.ts

```diff
@@ -87,5 +87,5 @@ export interface AzureResourceGraphQuery {
 export interface AzureMetricDimension {
   dimension: string;
   operator: string;
-  filter?: string;
+  filter?: string[];
```

Comment:
This is problematic. If we don't migrate existing queries, they will still send strings to the backend. If we for some reason don't want to migrate queries in the frontend but rely on backend to handle both, we should at least change typings here to `string | string[]`.